index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
-        such as HTTP, run the risk of being altered by an attacker such that
-        they may modify the JSON-LD <a>active context</a> in a way that
-        could compromise security. It is advised that any application that
-        depends on a remote context for mission critical purposes vet and
-        cache the remote context before allowing the system to use it.</p>
-      <p>Given that JSON-LD allows the substitution of long IRIs with short terms,
-        JSON-LD documents may expand considerably when processed and, in the worst case,
-        the resulting data might consume all of the recipient's resources. Applications
-        should treat any data with due skepticism.</p>
-      <p>As JSON-LD places no limits on the IRI schemes that may be used,
-        and vocabulary-relative IRIs use string concatenation rather than
-        IRI resolution, it is possible to construct IRIs that may be
-        used maliciously, if dereferenced.</p>
-    </dd>
-    <dt>Interoperability considerations:</dt>
-    <dd>Not Applicable</dd>
-    <dt>Published specification:</dt>
-    <dd>http://www.w3.org/TR/json-ld</dd>
-    <dt>Applications that use this media type:</dt>
-    <dd>Any programming environment that requires the exchange of
-      directed graphs. Implementations of JSON-LD have been created for
-      JavaScript, Python, Ruby, PHP, and C++.
-    </dd>
-    <dt>Additional information:</dt>
-    <dd>
-      <dl>
-        <dt>Magic number(s):</dt>
-        <dd>Not Applicable</dd>
-        <dt>File extension(s):</dt>
-        <dd>.jsonld</dd>
-        <dt>Macintosh file type code(s):</dt>
-        <dd>TEXT</dd>
-      </dl>
-    </dd>
-    <dt>Person &amp; email address to contact for further information:</dt>
-    <dd>Ivan Herman &lt;ivan@w3.org&gt;</dd>
-    <dt>Intended usage:</dt>
-    <dd>Common</dd>
-    <dt>Restrictions on usage:</dt>
-    <dd>N/A</dd>
-    <dt>Author(s):</dt>
-    <dd>Manu Sporny, Dave Longley, Gregg Kellogg, Markus Lanthaler, Niklas Lindström</dd>
-    <dt>Change controller:</dt>
-    <dd>W3C</dd>
-  </dl>
+    <p>Fragment identifiers used with <a href="#application-ld-json">application/ld+json</a>
+      are treated as in RDF syntaxes, as per
+      <a data-cite="RDF11-CONCEPTS#section-fragID">RDF 1.1 Concepts and Abstract Syntax</a>
+      [[RDF11-CONCEPTS]].</p>
 
-  <p>Fragment identifiers used with <a href="#application-ld-json">application/ld+json</a>
-    are treated as in RDF syntaxes, as per
-    <a data-cite="RDF11-CONCEPTS#section-fragID">RDF 1.1 Concepts and Abstract Syntax</a>
-    [[RDF11-CONCEPTS]].</p>
+    <p>This registration is an update to the original definition
+      for <a data-cite="JSON-LD10#application-ld-json">application/ld+json</a>
+      in [[JSON-LD10]].</p>
 
-  <p>This registration is an update to the original definition
-    for <a data-cite="JSON-LD10#application-ld-json">application/ld+json</a>
-    in [[JSON-LD10]].</p>
+    <section id="iana-examples" class="informative">
```

Comment:
Make section identifier unique, and add anchor for original identifier.

```suggestion
    <section id="media-type-examples" class="informative"><span id="iana-examples"></span>
```